README.md

```diff
@@ -27,6 +27,8 @@ available customizations and their default values.
         fzf/grep-command "grep -nrH"
         ;; If nil, the fzf buffer will appear at the top of the window
         fzf/position-bottom t
+        ;; If t, the fzf buffer will appear at the bottom of the frame
+        fzf/position-bottom nil
```

Comment:
@m-milek Don't you mean `fzf/position-bottom-of-frame` here instead?

Reply:
Yeah, fixed in e3f88ee26ce059f0f2309e13e5239a5d372dfbf4